fuzzing/fuzzer.go

```diff
@@ -343,21 +373,34 @@ func chainSetupFromCompilations(fuzzer *Fuzzer, testChain *chain.TestChain) (err
 		if len(fuzzer.contractDefinitions) == 1 {
 			fuzzer.config.Fuzzing.TargetContracts = []string{fuzzer.contractDefinitions[0].Name()}
 		} else {
-			return fmt.Errorf("missing target contracts (update fuzzing.targetContracts in the project config " +
-				"or use the --target-contracts CLI flag)"), nil
+			return fmt.Errorf("missing target contracts"), nil
 		}
 	}
 
-	// Loop for all contracts to deploy
+	// Concatenate the predeployed contracts and target contracts
+	// Ordering is important here (predeploys _then_ targets) so that you can have the same contract in both lists
+	contractsToDeploy := make([]string, 0)
```

Comment:
This can be statically allocated like `len(fuzzer.config.Fuzzing.PredeployedContracts) + len(fuzzer.config.Fuzzing.TargetContracts)`

Reply:
Yea the problem with the static allocation here is that you can't use the append anymore so you have to create two separate for loops to index into the array and set the different values (for target contracts and predeployed contracts).

Reply:
Also, with `PredeployedContracts` being a mapping, it may mildly increase complexity for the static allocation win.